packages/spotlight/src/container.js

```diff
@@ -272,6 +272,29 @@ const navigableFilter = (node, containerId) => {
 	return true;
 };
 
+/**
+ * Determines nodes that are owned by `node` based on `aria-owns`.
+ *
+ * @param   {Node}   node          Owner
+ *
+ * @returns {Node[]}               Array of owned nodes
+ * @memberof spotlight/container
+ * @private
+ */
+const getOwnedNodes = (node, selector) => {
+	// if node is document, it will not have getAttribute and therefore can't have aria-owns
+	const owns = node && node.getAttribute && node.getAttribute('aria-owns');
+	if (owns) {
+		const ids = owns.split(' ');
+		return ids
```

Comment:
I wonder if this could be simplified with `flatMap()`?  No need to actually do that, just wondering.

Reply:
I don't _think_ so. `flatMap` handles a map that returns an array but we're only working with single elements here.

Reply:
yeah, it's a little messy because of the operations.  With `flatMap()` you could return empty array to drop the unmatched without the `filter(Boolean)` but it can't be easily done without a temp variable.